sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeViewState.kt

```diff
@@ -0,0 +1,10 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import co.nimblehq.sample.compose.lib.IsLoading
+import co.nimblehq.sample.compose.model.UiModel
+
+data class HomeViewState(
```

Comment:
Do we need this `HomeViewState` anymore? 🤔

Reply:
@luongvo I'd like to get your opinion on this one too. 🙏🏻  Shall we combine all together in this `HomeViewState` or just do it separately like as-is.

Reply:
@lydiasama I think it's not related to what we need to achieve in this PR. Should we move it into a new discussion and a new task to refactor if needed?

I personally feel good with both, having `ViewState` is good too, but it's hard to manage screen states when having different data types, like `Survey` data loading state, `Header` data state, etc. This is kinda a big issue to discuss.

Reply:
@luongvo  I agree. 🙌🏻